.travis.yml

```diff
@@ -20,8 +20,6 @@ gemfile:
   - gemfiles/rails_51.gemfile
   - gemfiles/rails_50.gemfile
   - gemfiles/rails_42.gemfile
-  - gemfiles/rails_41.gemfile
-  - gemfiles/rails_40.gemfile
```

Comment:
Could you just leave them here let them fail? Removing will give users false hope and I would like to fix the failures rather than "hiding" them.

Reply:
Sure thing, I was going to start looking into the failures for another PR, but appreciate it if you have time for it :)